Android/app/src/main/java/com/example/boardgamesocial/DataClasses/Commons/Constants.java

```diff
@@ -0,0 +1,11 @@
+package com.example.boardgamesocial.DataClasses.Commons;
+
+public class Constants {
+
+    public static final String PREFS = "com.example.ecstasywebstore.prefs";
+    public static final String USER_TOKEN = "com.example.boardgamesocial.userToken";
+
+    public static final String GAME_MIN_PLAYER_TEXT = "Min players: ";
+    public static final String GAME_MAX_PLAYER_TEXT = "Max players: ";
+    public static final String GAME_OVERALL_PLAY_COUNT_TEXT = "Overall play count: ";
```

Comment:
These strings can be added to the strings.xml file for easy use and removed here if they are doing what I believe they are.

Reply:
It shouldn’t be an issue but need to look into it once